src/components/artifact/Launch.vue

```diff
@@ -16,12 +30,22 @@ const sharingKey = computed(() => {
 
     <div class="q-mb-sm">
       <q-btn
+        v-if="isJupyterHub"
         color="primary"
         label="Launch on Chameleon"
         :href="artifact.computed.get_chameleon_launch_url(version_slug, sharingKey)"
         target="_blank"
         class="full-width"
       />
+      <q-chip
+        v-else
+        color="grey"
+        text-color="white"
+        class="full-width justify-center cursor-inherit"
+        icon="open_in_new"
+      >
+        External Artifact
```

Comment:
the open icon here is a bit confusing, since this isn't a link right? The user just needs to read the description.

Reply:
removed